README.md

```diff
@@ -116,19 +116,44 @@ It is important for any code change to pass both unit and end-to-end tests. This
 
 ### Run End-to-End (E2E) Tests
 
-1. Run the token bridge UI locally on `http://localhost:3000/`
+1. Set up the Nitro node
+
+   1. Download and install [Docker](https://www.docker.com/)
 
-2. Set env vars (one time):
+   2. Check out the [Nitro repo](https://github.com/OffchainLabs/nitro)
 
-   1. At this folder's root,
+   3. Run:
 
       ```bash
-      $ cp ./packages/arb-token-bridge-ui/.e2e.env.sample ./packages/arb-token-bridge-ui/.e2e.env
+      $ yarn
       ```
 
-   2. In the newly created file, `.e2e.env`, update your `NEXT_PUBLIC_INFURA_KEY, PRIVATE_KEY_USER, etc` in the format mentioned in the file.
+   4. Run: (Make sure your Docker App is running)
+
+      ```bash
+      $ ./test-node.bash --init --no-blockscout
+      ```
+
+   5. When the Nitro node is up and running you should see logs like `sequencer_1` and `staker-unsafe_1` in the terminal. This can take up to 10 minutes
+
+2. In the root of the token bridge UI:
+
+   1. Run:
+
+   ```bash
+   $ cp ./packages/arb-token-bridge-ui/.e2e.env.sample ./packages/arb-token-bridge-ui/.e2e.env
+   ```
+
+   2. In the newly created file, `.e2e.env`, update your `NEXT_PUBLIC_INFURA_KEY, PRIVATE_KEY_USER, etc` in the format mentioned in the file. Some tests will require you to have 10+ deposits and withdrawals on Goerli in the transaction history. If your wallet doesn't have enough transactions, you can import and use this private key: `PRIVATE_KEY_USER=c791f745783e3545f83e7b4f0adcc0c2f591c5c8bc796e042c639824ec5d7bf0`
```

Comment:
The private key is a newly created wallet with no Goerli ETH but with transaction history populated